autopush/endpoint.py

```diff
@@ -261,7 +263,15 @@ def prepare(self):
         """Common request preparation"""
         if self.ap_settings.cors:
             self.set_header("Access-Control-Allow-Origin", "*")
-            self.set_header("Access-Control-Allow-Methods", self.cors_methods)
+            if self.cors_methods:
```

Comment:
Would we ever have occasion to not have values for any of these? If I recall, passing an empty header is equivalent to the header not being present.